src/main/java/uk/gov/hmcts/darts/common/entity/NodeRegisterEntity.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.darts.common.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = NodeRegisterEntity.TABLE_NAME)
+@Getter
+@Setter
+public class NodeRegisterEntity {
+
+    public static final String TABLE_NAME = "node_register";
+    public static final String NODE_GENERATOR = "nod_gen";
+    public static final String NODE_SEQUENCE = "nod_seq";
+    public static final String DEVICE_ID = "der_id";
+    public static final String COURTROOM_ID = "ctr_id";
+    public static final String NODE_ID = "node_id";
+
+    @Column(name = DEVICE_ID)
+    private int deviceId;
+
+    @ManyToOne
+    @JoinColumn(name = COURTROOM_ID)
```

Comment:
We should probably also specify `, nullable = false)` since the DDL already has NOT NULL?